docker/root/console-shell.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+cmd="php /hleb/console"
+
+for i in $@
+ do
+  cmd="$cmd $i"
+done
+
+$cmd
+
+/root/www-data.sh
```

Comment:
```suggestion
#!/bin/sh

cmd="php /hleb/console"

for i in $@
  do
    cmd="$cmd $i"
done

$cmd

sh /root/www-data.sh

```
Буквально несколько замечаний по code-style

В других sh-файлах после шебанга есть отступ в виде пропущенной строки. Я бы хотел, чтобы такое правило соблюдалось для всех sh-скриптов

Также я бы хотел, чтобы отступ перед началом строки во вложенных конструкциях был 2 пробела, вместо одного

Вызов одного скрипта из другого во всех остальных файлах начинается с указания имени программы. То есть перед `/root/www-data.sh` я бы хотел видеть `sh`, для единообразного стиля